accepted/future-releases/abstract-external-fields/feature-specification.md

```diff
@@ -0,0 +1,205 @@
+# Abstract Fields and External Fields
+
+Authors: lrn@google.com, eernst@google.com<br>
+Version: 1.0
+
+## Background and Motivation
+
+Dart allows abstract instance methods, getters and setters to be declared
+in classes and mixins.  It allows external functions, methods, getters, and
```

Comment:
I did that for many years, because various pieces of software wanted it (and Leslie Lamport said so ;-), then dropped it because some other pieces of software seemed to be more happy about a single space.

At this point I noted that the double space is used a lot (in at least 458 locations in the language repo) in `*.md` files, so I thought that the double space would be least disruptive in this context.

(Religious wars may proceed on demand ;-)

Reply:
(Deleted the double spaces, btw.)